tests/unit/amo/components/TestCategory.js

```diff
@@ -137,22 +139,23 @@ describe(__filename, () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
-  it('fetches categories and landing data when not yet loaded', () => {
+  it('fetches categories and landing data and sets a viewContext when not yet loaded', () => {
```

Comment:
Just FYI: we try to keep line length to 80 characters. It's not mandatory, just a nice-to-have for people like me who prefer to use horizontal splits. We haven't put a [lint rule](https://github.com/mozilla/addons-frontend/issues/3045) in place because no one has the courage to review the patch 🙂